src/world/area_hos/hos_00/A0BCF0.c

```diff
@@ -1,3 +1,10 @@
 #include "hos_00.h"
 
-INCLUDE_ASM(s32, "world/area_hos/hos_00/A0BCF0", func_80240000_A0BCF0);
+extern char D_800D9668[];
+
+ApiStatus func_80240000_A0BCF0(Evt* script, s32 isInitialCall) {
+    if (gGameStatusPtr->entryID == 3) {
+        sprintf(&D_800D9668, "hos_bg\0\0\0\0\0\0\0\0\0");
```

Comment:
I hope you can remove these nops?

Reply:
No, I specifically had to add them to get appropriate padding to happen :/

Reply:
😭